### ISSUE TITLE
Crashing when using  --include-package=win32api i.e. on an extension module

### ISSUE THREAD

**Issue by jekoie** [Nuitka/Nuitka#2062]
My Env:
system: win10 64bit system.
python: Python 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:34:34) [MSC v.1928 32 bit (Intel)] on win32
pywin32:   version 305(or 304)
nuitka:  
``` console
PS C:\Users\jekoi> python -m nuitka --version
1.4.7
Commercial: None
Python: 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:48:03) [MSC v.1928 64 bit (AMD64)]
Flavor: CPython Official
Executable: C:\Users\jekoi\AppData\Local\Programs\Python\Python38\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
Version C compiler: C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.32.31326\bin\HostX64\x64\cl.exe (cl 14.3).
```

compile cmd: `& C:/Python38_V32/python.exe  -m nuitka --noinclude-IPython-mode=nofollow --plugin-enable=pyqt5 --windows-icon-from-ico=./design/res/logo.ico --include-qt-plugins=sqldrivers,sensible,styles --show-progress --output-dir=dist --no-pyi-file --show-modules --include-data-dir=./setting=setting --include-package=websockets,uvicorn,certifi,requests,zmq,win32api,win32timezone  --show-modules-output=./dist/module.txt --show-memory --jobs=64 --lto=yes --prefer-source-code --python-flag=-S --windows-disable-console --nofollow-import-to=PIL,lib2to3,matplotlib,tornado,tkinter,unittest,zmq.tests --standalone main.py`

Error msg:
```console
Nuitka-Inclusion:INFO: Included 'zmq.utils.z85' as '<Node 'COMPILED_PYTHON_MODULE' with {'filename': 'C:\\Python38_V32\\lib\\site-packages\\zmq\\utils\\z85.py', 'module_name': <ModuleName zmq.utils.z85>, 'code_flags': ''}>'.
Nuitka-Inclusion:INFO: Checking top level path 'C:\Python38_V32\lib\site-packages\win32\win32api.pyd' 'None'.
Traceback (most recent call last):
  File "C:\Python38_V32\lib\site-packages\nuitka\__main__.py", line 151, in <module>
    main()
  File "C:\Python38_V32\lib\site-packages\nuitka\__main__.py", line 137, in main
    MainControl.main()
  File "C:\Python38_V32\lib\site-packages\nuitka\MainControl.py", line 942, in main
    main_module = _createMainModule()
  File "C:\Python38_V32\lib\site-packages\nuitka\MainControl.py", line 214, in _createMainModule
    Recursion.checkPluginPath(
  File "C:\Python38_V32\lib\site-packages\nuitka\importing\Recursion.py", line 335, in checkPluginPath
    if os.path.isfile(plugin_info[0]) or Importing.isPackageDir(plugin_info[0]):
  File "C:\Python38_V32\lib\genericpath.py", line 30, in isfile
    st = os.stat(path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType
```

**Comment by kayhayen** [comment]
Using `--include-package` on these should never be necessary. And it seems to not work. Whatever made you do these includes, probably it would be better to report those as issues, unless you really have code that requires this.

**Comment by kayhayen** [comment]
Also I wonder why you didn't get that I want `python -m nuitka --version` output. Writing up manually part of is is not needed or enough.

**Comment by kayhayen** [comment]
It seems that at some point, the code to include directories, lost its ability to refuse extension modules in there without a crash. It is of course wrong, because it is a module and not a package that `win32-api` points to. So this probably should have been `--include-module=win32api`.

Now it gives this:

```
Nuitka-Inclusion:WARNING: Failed to include from directory
Nuitka-Inclusion:WARNING: 'C:\Python310_64\lib\site-packages\win32\win32api.pyd'.
```

**Comment by kayhayen** [comment]
And for your case it now says this:

> FATAL: Error, the extension module 'win32api' is not a package, include it as a module instead.

**Comment by kayhayen** [comment]
Hm, for python modules, the code is more lenient, rejecting only extension modules for inclusion, so arguably this should either be mean all the way, or forgiving your error of wanting to treat it as a package here too. I guess it should be tolerant, because honestly who wants to lookup what it is.

**Comment by kayhayen** [comment]
Thanks for your report, this is fixed on the factory branch, which is a development version under rapid development. You can try it out by going here: https://nuitka.net/doc/factory.html 

Feedback if this is working is very welcome, just please do not share plans of doing it, but rather confirmations or denials of it working.

**Comment by jekoie** [comment]
Hi, I had add more detail nuitka version. I prefer like to "--include-package" for sometime I worry the app will run error when I lost some sub-modules.

**Comment by kayhayen** [comment]
Yes, you can of course have a bigger and bloated binary if you are OK with that. I invite reports and esp. fixes with the tools in the post series at https://nuitka.net/posts/nuitka-package-config-kickoff.html so things work for everybody optimally and out of the box, but I understand not everybody cares.

That said, the fix is now on develop and will be part of 1.5 release.

**Comment by kayhayen** [comment]
This is part of the stable release 1.5 that I just made.
